update/delta/delta_1.3.6_add_status_project_execution.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO qwat_vl.status (id, active, functional, value_en, value_fr, value_ro, description_en, description_fr, description_ro) VALUES (1308, false, false, 'project in execution',        'projet en execution',       'Proiect în execuție',      '', 'L''objet est un projet validé en cours de réalisation', 'Obiectul face parte dintr-un proiect care este în curs de execuție');
```

Comment:
Can you uppercase for FR and EN. And add long description for EN ?

Reply:
Sure about the en description. For the uppercase part.. I've uppercased for all the values (not only 1308).

Commens addressed.